config/redirects.yml

```diff
@@ -3,7 +3,7 @@ plugins:
     redirect_maps:
       contributing/about.md: contributing/README.md
       eventing/broker/kafka-broker/kafka-configmap.md: eventing/configuration/kafka-channel-configuration.md
-      eventing/broker/create-mtbroker.md: eventing/brokers/create-mtbroker.md
+      eventing/broker/create-mtbroker.md: eventing/brokers/create-broker.md
```

Comment:
Should this point to the `docs/eventing/brokers/broker-types/mt-channel-based-broker/README.md` file now?

Reply:
We could. Just not sure what is more confusing, as before this was the "Create Broker" section under "Brokers" and from the title not necessarily only related to MTCBB :shrug: